redgenes/gene_annotations.py

```diff
@@ -370,14 +382,23 @@ def annotation_pipeline(df, tmpdir, kofamscan_profile, kofamscan_kolist):
 def main():
     initialize_db()
 
-    md_tsv = "./redgenes/tests/data/md_gordon_2.tsv"
-    md_df = extract_md_info(md_tsv)
+    # md_tsv = "./redgenes/tests/data/md_gordon_2.tsv"
+    md_str = sys.argv[1]
+    md_df = extract_md_info(md_str)
+
+    # start_time = time.time()
 
     ko_profile = "/projects/greengenes2/20231117_annotations_prelim/kofam_scan/profiles/test_subset.hal"
     ko_kolist = (
         "/projects/greengenes2/20231117_annotations_prelim/kofam_scan/ko_list_subset"
     )
-    annotation_pipeline(md_df, "./test_out", ko_profile, ko_kolist)
+    annotation_pipeline(md_df, "/panfs/y1weng/test_out2", ko_profile, ko_kolist)
```

Comment:
```suggestion
    annotation_pipeline(md_df, working_directory, ko_profile, ko_kolist)
```

Normally, I would advise using `tempfile` in a context manager around its use. However that would be a bit larger of a refactor, and is just a nice-to-have